js/layout-javascript/small-h-card-code.js

```diff
@@ -99,248 +101,258 @@ DynamicList.prototype.attachObservers = function() {
-        });
-        return;
-      }
+        if (!_.get(_this, 'data.editEntryLinkAction.page')) {
+          Fliplet.UI.Toast({
+            title: 'Link not configured',
+            message: 'Form not found. Please check the component\'s configuration.'
+          });
+          return;
+        }
 
-      var entryID = $(this).parents('.small-h-card-detail-overlay').find('.small-h-card-list-detail-content-scroll-wrapper').data('entry-id');
+        var entryID = $(this).parents('.small-h-card-detail-overlay').find('.small-h-card-list-detail-content-scroll-wrapper').data('entry-id');
 
-      _this.data.editEntryLinkAction.query = '?dataSourceEntryId=' + entryID;
+        _this.data.editEntryLinkAction.query = '?dataSourceEntryId=' + entryID;
 
-      try {
-        var navigate = Fliplet.Navigate.to(_this.data.editEntryLinkAction);
+        try {
+          var navigate = Fliplet.Navigate.to(_this.data.editEntryLinkAction);
 
-        if (navigate instanceof Promise) {
-          navigate
-            .catch(function(error) {
-              Fliplet.UI.Toast(error, {
-                message: 'Error editing entry'
+          if (navigate instanceof Promise) {
+            navigate
+              .catch(function(error) {
+                Fliplet.UI.Toast(error, {
+                  message: 'Error editing entry'
+                });
               });
-            });
+          }
+        } catch (error) {
+          Fliplet.UI.Toast(error, {
+            message: 'Error editing entry'
+          });
         }
-      } catch (error) {
-        Fliplet.UI.Toast(error, {
-          message: 'Error editing entry'
-        });
       }
     })
-    .on('click', '.dynamic-list-delete-item', function() {
-
-      var _that = $(this);
-      var entryID = $(this).parents('.small-h-card-detail-overlay').find('.small-h-card-list-detail-content-scroll-wrapper').data('entry-id');
-      var options = {
-        title: 'Are you sure you want to delete the list entry?',
-        labels: [
-          {
-            label: 'Delete',
-            action: function (i) {
-              _that.text('Deleting...').addClass('disabled');
-
-              // Run Hook
-              Fliplet.Hooks.run('flListDataBeforeDeleteEntry', {
-                instance: _this,
-                entryId: entryID,
-                config: _this.data,
-                id: _this.data.id,
-                uuid: _this.data.uuid,
-                container: _this.$container
-              })
-                .then(function() {
-                  if (_this.data.deleteData && typeof _this.data.deleteData === 'function') {
-                    return _this.data.deleteData(entryID);
-                  }
-
-                  return _this.deleteEntry(entryID);
+    .on('click keydown', '.dynamic-list-delete-item', function(event) {
+      if (_this.Utils.Accessibility.accessibilityDetails(event, $(this))) {
```

Comment:
1. You're sending a variable that isn't used.
2. Change the function name and namespace.
3. Use a `if (flag) { return } /* Continue  */` logic to reduce the need for indentation

```suggestion
      if (!_this.Utils.Event.isExecEvent(event)) {
        return;
      }
```

Reply:
Number 2 was not actioned

Reply:
Number 2 was not actioned correctly. The namespace is incorrect.